modules/RTC/DataChannels.js

```diff
@@ -247,4 +239,37 @@ DataChannels.prototype._some = function (callback, thisArg) {
     }
 };
 
+/**
+ * Sends passed object via the first found open datachannel
+ * @param jsonObject {object} the object that will be sent
+ */
+DataChannels.prototype.send = function (jsonObject) {
```

Comment:
@bgrozev But we are checking if the channel is open before sending. What is the use case you are referring to?